recipes/_common_scheduler.rb

```diff
@@ -37,10 +37,10 @@
 
 ruby_block 'set thermos_executor flags' do
   block do
-    node.default['aurora']['scheduler']['app_config']['thermos_executor_flags'] =
+    node.default['aurora']['scheduler']['app_config']['thermos_executor_flags'] <<
       if node['aurora']['thermos']['announcer_enable']
         [
-          '--announcer-enable',
+          ' --announcer-enable',
```

Comment:
Initially it was to be able to append when there is already something in `node.default['aurora']['scheduler']['app_config']['thermos_executor_flags']`. There are probably better, and non-ugly ways to solve that :smile: